src/yetibot/core/commands/karma/specs.clj

```diff
@@ -1,33 +1,32 @@
 (ns yetibot.core.commands.karma.specs
   (:require [clojure.spec.alpha :as s]))
 
-;; Common specs we could relocate
 (s/def ::name string?)
 (s/def ::id string?)
 (s/def ::user (s/keys :req-un [::name ::id]))
 
-(s/def ::adapter #{:graphql :irc :slack})
```

Comment:
What's the reasoning on removing chat-source and adapter specs? Just curious.

Reply:
I was using adapter details to drive formatting decisions.  After removing that formatting logic, the adapter key value was no longer used anywhere, so I cleaned it out of the spec.  I figured it's better to prune it than to leave in place, unused. Easy enough to add back in if we want to generalize the specs a bit and pull most of them out of Karma.